### PR TITLE
Release notes 2020.02

### PR DESCRIPTION
**Pull request by rachael-phillips**
Please review these 2020.02 release notes

**What**: Closes #

**Additional issues**:

### CODE REVIEW

**Comment by kmcfaul** [comment]
The beta SimpleList component was merged earlier today if we want to add that to the release notes: #3645

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,6 +4,61 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.02 release notes (2020-02-18)
+Packages released:
+-
+
+### Components:
+- **ContextSelector:** Updated signature for context selector ([#3697](https://github.com/patternfly/patternfly-react/pull/3697))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: This structure  does not match previous release notes.  "Updated signature...." should be on a new line and have a bullet.

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -4,6 +4,61 @@ section: overview
...
+  - Phrased non production components as beta ([#3663](https://github.com/patternfly/patternfly-react/pull/3663))
+  - Linted react topology ([#3721](https://github.com/patternfly/patternfly-react/pull/3721))
+- **Cypress:** Parallelized tests and run in PRs ([#3627](https://github.com/patternfly/patternfly-react/pull/3627))
+- **Docs:** 
+  - Updated table property descriptions ([#3745](https://github.com/patternfly/patternfly-react/pull/3745))
+  - Updated node version in readme ([#3741](https://github.com/patternfly/patternfly-react/pull/3741))
+  - Bumped gatsby ([#3657](https://github.com/patternfly/patternfly-react/pull/3657))
+  - Merged branch gatsby-theme-patternfly-org ([#3651](https://github.com/patternfly/patternfly-react/pull/3651))
+  - Merged patternfly-org branch ([#3619](https://github.com/patternfly/patternfly-react/pull/3619))
+- **Eslint:** Added react-hooks to eslint-plugin-patternfly-react ([#3636](https://github.com/patternfly/patternfly-react/pull/3636))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: Maybe move this bullet point under "Linting" below.

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -4,6 +4,61 @@ section: overview
...
+- **Github:** Fixed broken link in issue template ([#3634](https://github.com/patternfly/patternfly-react/pull/3634))
+- **Icons:** Made change to use absolute import paths ([#3517](https://github.com/patternfly/patternfly-react/pull/3517))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: I would probably put this under "Chore"

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -4,6 +4,61 @@ section: overview
...
+- Added inline edit to table ([#3058](https://github.com/patternfly/patternfly-react/issues/3058))
+- Reverted IHeaderRow interface change ([#3746](https://github.com/patternfly/patternfly-react/pull/3746))
+- Fixed that column functions are not always equal ([#3612](https://github.com/patternfly/patternfly-react/pull/3612))
+- Added support for truncated column headers ([#3729](https://github.com/patternfly/patternfly-react/pull/3729))
+
+### Other:
+- **Build:**
+  - Updated circleci config for pricing ([#3654](https://github.com/patternfly/patternfly-react/pull/3654))
+  - Made change to use machine instance ([#3712](https://github.com/patternfly/patternfly-react/pull/3712))
+  - Fixed typo, added md lint to circleci ([#3692](https://github.com/patternfly/patternfly-react/pull/3692))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: I would capitalize CircleCi and Gatsby since they are product names.

File: RELEASE-NOTES.md (hunk 1 above)
Comment (tlabaj) [inline RELEASE-NOTES.md]: WE have been going for sentence case with the names. So this should be "Context selector".  Same applies for the other components below

Comment (rachael-phillips) [reply RELEASE-NOTES.md]: Thank you!

File: RELEASE-NOTES.md (hunk 4 above)
Comment (rachael-phillips) [reply RELEASE-NOTES.md]: Thank you!

File: RELEASE-NOTES.md (hunk 3 above)
Comment (rachael-phillips) [reply RELEASE-NOTES.md]: Which one?

Comment (rachael-phillips) [reply RELEASE-NOTES.md]: Icons?

File: RELEASE-NOTES.md (hunk 1 above)
Comment (rachael-phillips) [reply RELEASE-NOTES.md]: Thank you!

File: RELEASE-NOTES.md (hunk 2 above)
Comment (rachael-phillips) [reply RELEASE-NOTES.md]: Will do!

File: RELEASE-NOTES.md (hunk 3 above)
Comment (tlabaj) [reply RELEASE-NOTES.md]: yes

Comment (tlabaj) [reply RELEASE-NOTES.md]: Github can probably go under "chore" as well.

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -4,6 +4,70 @@ section: overview
...
+- [@patternfly/react-table: 2.27.5](https://www.npmjs.com/package/@patternfly/react-table/v/2.27.5)
+- [@patternfly/react-tokens: 2.8.2](https://www.npmjs.com/package/@patternfly/react-tokens/v/2.8.2)
+- [@patternfly/react-topology: 2.14.5](https://www.npmjs.com/package/@patternfly/react-topology/v/2.14.5)
+- [@patternfly/react-virtualized-extension: 1.4.6](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/1.4.6)
+- [@patternfly/react-icons: 3.15.0](https://www.npmjs.com/package/@patternfly/react-icons/v/3.15.0)
+
+### Components:
+- **Context selector:** Updated signature for context selector ([#3697](https://github.com/patternfly/patternfly-react/pull/3697))
+- **Data toolbar:** Updated html structure to match core ([#3701](https://github.com/patternfly/patternfly-react/pull/3710))
+- **Filterable table:** Fixed the TextInput type in filter table demo ([#3652](https://github.com/patternfly/patternfly-react/pull/3652))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: This should be moved to the "Other" section under "Demos"

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -4,6 +4,70 @@ section: overview
...
+- **Popover:** Added min-width override property ([#3601](https://github.com/patternfly/patternfly-react/pull/3601))
+- **Radio:** Added support for optional description ([#3621](https://github.com/patternfly/patternfly-react/pull/3621))
+- **Select:** Handled undefined default for checkbox select ([#3711](https://github.com/patternfly/patternfly-react/pull/3711))
+- **Simple list:** Added Simple list component ([#3645](https://github.com/patternfly/patternfly-react/pull/3645))
+- **Slider:** Added limit for lowest possible value of slider ([#3660](https://github.com/patternfly/patternfly-react/pull/3660))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: This should be moved to PF3 section.

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -4,6 +4,70 @@ section: overview
...
+- **Wizard:** Added typeto wizard context ([#3572](https://github.com/patternfly/patternfly-react/pull/3572))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: I think there was a typo in his commit message. This should say "type to".

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -4,6 +4,70 @@ section: overview
...
+
+### Catalog view extension
+- **Catalog tile:** 
+  - Adjusted max width of header image ([#3628](https://github.com/patternfly/patternfly-react/pull/3628))
+  - Reduced padding between badge and logo ([#3644](https://github.com/patternfly/patternfly-react/pull/3644))
+  - Fixed styles that stopped working ([#3635](https://github.com/patternfly/patternfly-react/pull/3635))
+  - Fixed linting errors ([#3714](https://github.com/patternfly/patternfly-react/pull/3714))
+
+### Charts
+- **React charts:** Fixed linting errors ([#3713](https://github.com/patternfly/patternfly-react/pull/3713))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: I wonder if we should put this under "Linting" with all the other PRs that fixed linting issues?

Comment (rachael-phillips) [reply RELEASE-NOTES.md]: I agree - will do

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -4,6 +4,70 @@ section: overview
...
+### Other:
+- **Build:**
+  - Updated Circle CI config for pricing ([#3654](https://github.com/patternfly/patternfly-react/pull/3654))
+  - Made change to use machine instance ([#3712](https://github.com/patternfly/patternfly-react/pull/3712))
+  - Fixed typo, added md lint to Circle CI ([#3692](https://github.com/patternfly/patternfly-react/pull/3692))
+- **Chore:** 
+  - Phrased non production components as beta ([#3663](https://github.com/patternfly/patternfly-react/pull/3663))
+  - Linted React topology ([#3721](https://github.com/patternfly/patternfly-react/pull/3721))
+  - For icons made change to use absolute import paths ([#3517](https://github.com/patternfly/patternfly-react/pull/3517))
+- **Cypress:** Parallelized tests and run in PRs ([#3627](https://github.com/patternfly/patternfly-react/pull/3627))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: Can we change this to "CI" to match our last release notes?

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -4,6 +4,70 @@ section: overview
...
+- **Docs:** 
+  - Updated table property descriptions ([#3745](https://github.com/patternfly/patternfly-react/pull/3745))
+  - Updated node version in readme ([#3741](https://github.com/patternfly/patternfly-react/pull/3741))
+  - Bumped Gatsby ([#3657](https://github.com/patternfly/patternfly-react/pull/3657))
+  - Merged branch gatsby-theme-patternfly-org ([#3651](https://github.com/patternfly/patternfly-react/pull/3651))
+  - Merged patternfly-org branch ([#3619](https://github.com/patternfly/patternfly-react/pull/3619))
+- **Github:** Fixed broken link in issue template ([#3634](https://github.com/patternfly/patternfly-react/pull/3634))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: Maybe just move this under chore.  I looked back at our old release notes and that is what we had the template work under.

File: RELEASE-NOTES.md (hunk 9 above)
Comment (rachael-phillips) [reply RELEASE-NOTES.md]: Yes :D

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -4,6 +4,69 @@ section: overview
...
+- **Build:**
+  - Updated CircleCi config for pricing ([#3654](https://github.com/patternfly/patternfly-react/pull/3654))
+  - Made change to use machine instance ([#3712](https://github.com/patternfly/patternfly-react/pull/3712))
+  - Fixed typo, added md lint to CircleCi ([#3692](https://github.com/patternfly/patternfly-react/pull/3692))
+- **Chore:** 
+  - Phrased non production components as beta ([#3663](https://github.com/patternfly/patternfly-react/pull/3663))
+  - Linted React topology ([#3721](https://github.com/patternfly/patternfly-react/pull/3721))
+  - For icons made change to use absolute import paths ([#3517](https://github.com/patternfly/patternfly-react/pull/3517))
+  - Fixed broken link in issue template ([#3634](https://github.com/patternfly/patternfly-react/pull/3634))
+- **Ci:** Parallelized tests and run in PRs ([#3627](https://github.com/patternfly/patternfly-react/pull/3627))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: can we move this and the one below to their own bulleted line as we have been doing in other release notes?

File: RELEASE-NOTES.md [hunk 12]
```diff
@@ -4,6 +4,69 @@ section: overview
...
+- [@patternfly/react-inline-edit-extension: 2.17.5](https://www.npmjs.com/package/@patternfly/react-inline-edit-extension/v/2.17.5)
+- [@patternfly/react-styles: 3.7.2](https://www.npmjs.com/package/@patternfly/react-styles/v/3.7.2)
+- [@patternfly/react-table: 2.27.5](https://www.npmjs.com/package/@patternfly/react-table/v/2.27.5)
+- [@patternfly/react-tokens: 2.8.2](https://www.npmjs.com/package/@patternfly/react-tokens/v/2.8.2)
+- [@patternfly/react-topology: 2.14.5](https://www.npmjs.com/package/@patternfly/react-topology/v/2.14.5)
+- [@patternfly/react-virtualized-extension: 1.4.6](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/1.4.6)
+- [@patternfly/react-icons: 3.15.0](https://www.npmjs.com/package/@patternfly/react-icons/v/3.15.0)
+
+### Components:
+- **Context selector:** Updated signature for context selector ([#3697](https://github.com/patternfly/patternfly-react/pull/3697))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: Same comment here about the format and moving the PRs to there own bullet.

File: RELEASE-NOTES.md [hunk 13]
```diff
@@ -4,6 +4,69 @@ section: overview
...
+- **Data toolbar:** Updated html structure to match core ([#3701](https://github.com/patternfly/patternfly-react/pull/3710))
+- **Popover:** Added min-width override property ([#3601](https://github.com/patternfly/patternfly-react/pull/3601))
+- **Radio:** Added support for optional description ([#3621](https://github.com/patternfly/patternfly-react/pull/3621))
+- **Select:** Handled undefined default for checkbox select ([#3711](https://github.com/patternfly/patternfly-react/pull/3711))
+- **Simple list:** Added Simple list component ([#3645](https://github.com/patternfly/patternfly-react/pull/3645))
+- **Slider:** Added limit for lowest possible value of slider ([#3660](https://github.com/patternfly/patternfly-react/pull/3660))
```
Comment (tlabaj) [inline RELEASE-NOTES.md]: This should be moved to PF3 section under a "Slider". (See previous release notes)

**Pull request merged**